pom.xml

```diff
@@ -1888,7 +1888,8 @@
                     <overWriteSnapshots>false</overWriteSnapshots>
                     <overWriteIfNewer>true</overWriteIfNewer>
                     <excludeGroupIds>org.apache.hadoop,org.apache.hbase,org.apache.hive,com.google.protobuf,\
-                      org.datanucleus,asm,org.apache.spark,slf4j-log4j12,org.codehaus.jackson</excludeGroupIds>
+                      org.datanucleus,asm,org.apache.spark,slf4j-log4j12,org.codehaus.jackson,\
+                      org.apache.tez</excludeGroupIds>
```

Comment:
The direct dependency is introduced by  [https://github.com/caskdata/cdap/pull/6717/files#diff-600376dffeb79835ede4a0b285078036R114]

conf.set(TezConfiguration.TEZ_CLUSTER_ADDITIONAL_CLASSPATH_PREFIX, additionalClassPath);